### ISSUE TITLE
Getting gradle sync error when using with ios

### ISSUE THREAD

**Issue by Tlaster** [icerockdev/moko-resources#355]
Get error when using with moko resources
```
> Failed to notify project evaluation listener.
   > Cannot change attributes of dependency configuration ':sample:common:iosArm64ApiElements' after it has been resolved
   > Cannot change attributes of dependency configuration ':sample:common:iosX64ApiElements' after it has been resolved
```
It happens when after adding moko resources, my gradle setup like this
```
plugins {
    kotlin("multiplatform")
    id("dev.icerock.mobile.multiplatform-resources") version "0.20.1"
}
kotlin {
    ios()
    sourceSets {
        val commonMain by getting
        val commonTest by getting
    }
}
multiplatformResources {
    multiplatformResourcesPackage = "moe.tlaster.common"
}
```
Gradle version: 7.4.2
Kotlin version: 1.6.21

Am I doing something wrong?

**Comment by Alex009** [comment]
can you create project where this bug reproduced and send here?

**Comment by Tlaster** [comment]
It's here https://github.com/Tlaster/moko-spotless-conflict
I think it might have a conflict with spotless, I can not apply moko resource with spotless at the same time.

**Comment by Alex009** [comment]
@Tlaster thanks, your project helps. i found cause of error and fix will be in next release

**Comment by asarazan** [comment]
Hi there! Noticed this fix went out back in July, and was hoping it would be possible to get a patch version published with the Spotless fix?

**Comment by Alex009** [comment]
will be released in 0.21.0
